docs/03-03-examples.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Kyma CLI usage examples
+type: Details
+---
+
+The following examples show how to use Kyma CLI.
+
+- Install Kyma with Minikube on Mac:
+
+    ```bash
+    kyma provision minikube
+    kyma install
+    ```
+
+- Install Kyma with Minikube on Windows:
```

Comment:
I think we can merge this with the previous example. "Install Kyma with Minikube on Mac or Windows:"

Reply:
True 👍